### PR TITLE
{bio}[gompi/2021b] HH-suite v3.3.0

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/h/HH-suite/HH-suite-3.3.0-gompi-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    'files': ['bin/%s' % x for x in _binaries] +
+             ['scripts/%s' % y for y in _scriptfiles],
+    'dirs': ['data', 'scripts']
+}
+
+modextrapaths = {
+    'PATH': 'scripts',
+    'PERL5LIB': 'scripts',
+}
+
```
Comment (bedroge) [inline easybuild/easyconfigs/h/HH-suite/HH-suite-3.3.0-gompi-2021b.eb:65]: Recent easyconfigs of HH-suite contain the following (see for instance https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/h/HH-suite/HH-suite-3.3.0-gompi-2021a.eb):

```suggestion

modextravars = {
    'HHLIB': '%(installdir)s',
}

```

Could you please add this here as well?

File: easybuild/easyconfigs/h/HH-suite/HH-suite-3.3.0-gompi-2021b.eb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+]
+
+fix_perl_shebang_for = ['scripts/*pl']
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in _binaries] +
+             ['scripts/%s' % y for y in _scriptfiles],
+    'dirs': ['data', 'scripts']
+}
+
```
Comment (bedroge) [inline easybuild/easyconfigs/h/HH-suite/HH-suite-3.3.0-gompi-2021b.eb:60]: And could we also include some sanity check commands here for a few (or maybe even all the ones in `_binaries`?) binaries?

Comment (bedroge) [reply easybuild/easyconfigs/h/HH-suite/HH-suite-3.3.0-gompi-2021b.eb:60]: This seems to work for me:
```suggestion

sanity_check_commands = ['%s -h' % x for x in _binaries]

```

Comment (fizwit) [reply easybuild/easyconfigs/h/HH-suite/HH-suite-3.3.0-gompi-2021b.eb:60]: will add sanity_check_commands - There are 51 binaries

File: easybuild/easyconfigs/h/HH-suite/HH-suite-3.3.0-gompi-2021b.eb (hunk 1 above)
Comment (fizwit) [reply easybuild/easyconfigs/h/HH-suite/HH-suite-3.3.0-gompi-2021b.eb:65]: sure

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3505.doduo.os - Linux RHEL 8.4, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/4204d73271f20c214db4d6fdfad5f2cb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3100.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/c3eec7472351bd1bc4d8d699dfebc3ed for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @fizwit!

**Pull request merged**